resident/resident-service/src/main/java/io/mosip/resident/entity/ResidentTransactionEntity.java

```diff
@@ -2,10 +2,7 @@
 
 import java.time.LocalDateTime;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
+import jakarta.persistence.*;
```

Comment:
remove the usage of * please specify the classes that are required to be import.

Reply:
Corrected